From 06c9d7aec1b660b7e2b144a9a9af23ac093ff2e2 Mon Sep 17 00:00:00 2001
From: Laio Oriel Seman <laioseman@gmail.com>
Date: Mon, 14 Oct 2024 14:24:14 -0300
Subject: [PATCH 1/2] patch

---
 src/lp_data/HighsRuntimeOptions.h | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/lp_data/HighsRuntimeOptions.h b/src/lp_data/HighsRuntimeOptions.h
index c4b7ecc1e..06057091b 100644
--- a/src/lp_data/HighsRuntimeOptions.h
+++ b/src/lp_data/HighsRuntimeOptions.h
@@ -88,13 +88,6 @@ bool loadOptions(const HighsLogOptions& report_log_options, int argc,
       std::cout << cxx_options.help({""}) << std::endl;
       exit(0);
     }
-    if (result.count("version")) {
-      std::cout << "HiGHS version " << HIGHS_VERSION_MAJOR << "."
-                << HIGHS_VERSION_MINOR << "." << HIGHS_VERSION_PATCH;
-      std::cout << " Githash " << HIGHS_GITHASH << ". ";
-      std::cout << kHighsCopyrightStatement << std::endl;
-      exit(0);
-    }
     if (result.count(kModelFileString)) {
       auto& v = result[kModelFileString].as<std::vector<std::string>>();
       if (v.size() > 1) {
-- 
2.47.0

From 183a542d248d19c1d59235305ed6ce0413815a7c Mon Sep 17 00:00:00 2001
From: Laio Oriel Seman <laioseman@gmail.com>
Date: Mon, 14 Oct 2024 14:28:22 -0300
Subject: [PATCH 2/2] patch

---
 src/io/HighsIO.cpp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/io/HighsIO.cpp b/src/io/HighsIO.cpp
index 38fbd6b40..300ad5856 100644
--- a/src/io/HighsIO.cpp
+++ b/src/io/HighsIO.cpp
@@ -22,13 +22,6 @@
 
 void highsLogHeader(const HighsLogOptions& log_options,
                     const bool log_githash) {
-  const std::string githash_string(HIGHS_GITHASH);
-  const std::string githash_text =
-      log_githash ? " (git hash: " + githash_string + ")" : "";
-  highsLogUser(log_options, HighsLogType::kInfo,
-               "Running HiGHS %d.%d.%d%s: %s\n", (int)HIGHS_VERSION_MAJOR,
-               (int)HIGHS_VERSION_MINOR, (int)HIGHS_VERSION_PATCH,
-               githash_text.c_str(), kHighsCopyrightStatement.c_str());
 }
 
 std::array<char, 32> highsDoubleToString(const double val,
-- 
2.47.0

